react/components/Slider.js

```diff
@@ -1,7 +1,393 @@
 import React from 'react'
+import React, { Children, Component, cloneElement } from 'react'
+import debounce from 'debounce'
+import PropTypes from 'prop-types'
+import { resolveSlidesNumber } from '../utils'
 
-const Slider = () => (
-  <div>Slider works</div>
-)
+class Slider extends Component {
+  static propTypes = {
+    resizeDebounce: PropTypes.number,
+    onChangeCurrentSlide: PropTypes.func.isRequired,
+    duration: PropTypes.number,
+    easing: PropTypes.string,
+    perPage: PropTypes.oneOfType([PropTypes.number, PropTypes.object]),
+    draggable: PropTypes.bool,
+    threshold: PropTypes.number,
+    howManySlides: PropTypes.number,
```

Comment:
same here, needs a less convoluted name, don't know what to rename to.

Reply:
`slidesNumber`, `slidesLength`, `slidesN`, `howManySlidesCanIGetYou?`...

Reply:
This prop doesn't exist anymore. I forgot to remove it from `propTypes`